pkgs/os-specific/linux/kvmfr/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, stdenv, fetchFromGitHub, kernel, kmod, looking-glass-client }:
+
+stdenv.mkDerivation rec {
+  pname = "kvmfr-${version}-${kernel.version}";
```

Comment:
`mkDerivation` already derives `name` as `${pname}-${version}`, so it is not required here. I'd also suspect that `${kernel.version}` is not required here, as it is already bound to the kernel version by having it as an input.

Reply:
Good to know. Changed `pname` to `"kvmfr"`